sklearn/cluster/_optics.py

```diff
@@ -664,27 +664,27 @@ def _extend_region(steep_point, xward_point, start, min_samples):
 
     Parameters
     ----------
-    steep_point : bool array, shape (n_samples)
+    steep_point : bool array of shape (n_samples,)
```

Comment:
When specifying the data type of an `ndarray`, use e.g. `dtype=np.int32` after defining the shape. In this case, `ndarray of shape (n_samples,), dtype=np.bool`.

Reply:
ok